src/hooks/useTokenPriceUsd.ts

```diff
@@ -1,16 +1,17 @@
 import { useQuery } from "@tanstack/react-query"
 import { Address } from "wagmi"
 
+import { getCoinGeckoPrice } from "@/lib/api/pricer/getCoinGeckoPrice"
 import { getGlpPrice } from "@/lib/api/pricer/getGlpPrice"
 import { getLlamaPrice } from "@/lib/api/pricer/getLlamaPrice"
 import { queryKeys } from "@/lib/helpers"
 import { useActiveChainId } from "@/hooks"
 
-import { glpTokenAddress } from "@/constant/addresses"
+import { fcGlpTokenAddress, glpTokenAddress } from "@/constant/addresses"
 
 const customPricers: Record<Address, () => Promise<number>> = {
   [glpTokenAddress]: getGlpPrice,
-  ["0xBDF9001c5d3fFc03AB6564CA28E530665594dfF7"]: getGlpPrice,
+  [fcGlpTokenAddress]: getGlpPrice,
```

Comment:
Is this necessary?

Reply:
Nope, it was introduced by the concentrator - it's not related to this issue. As I started from the main branch, that's why - I saw I hardcoded the address here, so I changed it.